nrf_802154_sl/Kconfig

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+if NRF_802154_RADIO_DRIVER
```

Comment:
Think this should have an additional level, similar to BLE controller and Bluetooth Link Layer.
![image](https://user-images.githubusercontent.com/9802583/82906953-4917d280-9f66-11ea-9f43-fe3e396eb926.png)